lbrynet/db_migrator/dbmigrator.py

```diff
@@ -1,18 +1,14 @@
 import logging
-# import os
 
 
 def migrate_db(db_dir, start, end):
     current = start
-    old_dirs = []
-    # if os.name == "nt":
-    #     return old_dirs
-    # while current < end:
-    #     if current == 0:
-    #         from lbrynet.db_migrator.migrate0to1 import do_migration
-    #         old_dirs.append(do_migration(db_dir))
-    #         current += 1
-    return old_dirs
+    while current < end:
+        if current == 1:
+            from lbrynet.db_migrator.migrate1to2 import do_migration
+            do_migration(db_dir)
+            current += 1
+    return None
```

Comment:
I don't understand this logic.  What if current = 2 and end = 3, this will loop forever.